projects/ccf-rui/src/app/shared/components/details-label/details-label.component.scss

```diff
@@ -0,0 +1,3 @@
+.label-container {
```

Comment:
Wrap everything in a `:host { ... }` block. Then you can also style the element itself directly. I.e. no need for top level div wrappers etc.